addToCart/style.css

```diff
@@ -0,0 +1,110 @@
+*{
```

Comment:
this is better if applied to html and body rather than to everything

Reply:
i usually do like to reset the padding and the margin of everything, is it a problem if i'm doing it like so ? i hate when things mess up beacause of some default padding/margin